pom.xml

```diff
@@ -201,7 +201,7 @@ flexible messaging model and an intuitive client API.</description>
     <json-smart.version>2.4.10</json-smart.version>
     <opensearch.version>1.2.4</opensearch.version>
     <elasticsearch-java.version>8.12.1</elasticsearch-java.version>
-    <debezium.version>1.9.7.Final</debezium.version>
+    <debezium.version>2.6.1.Final</debezium.version>
```

Comment:
According to https://debezium.io/ , latest stable Debezium release series is 2.7.x . Could we upgrade directly to 2.7.x instead?

Reply:
From 2.6.2.Final onwards the postgres tester is not working. The consumer is expecting a message event just after the insertion into the table but the source is taking some time to process the event. I have tested this locally as well.
I think it is due to PostgreSQL offset flush race condition. More details are here:
1. [Release Notes](https://debezium.io/blog/2024/05/30/debezium-2-6-2-final-released/)
2. [DBZ-7816](https://issues.redhat.com/browse/DBZ-7816)

Reply:
> From 2.6.2.Final onwards the postgres tester is not working.

[DBZ-7816](https://issues.redhat.com/browse/DBZ-7816) is fixed in 2.6.2.Final. Perhaps something needs to be adjusted in the Pulsar Debezium integration?